src/IrrationalConstants.jl

```diff
@@ -26,8 +26,12 @@ export
     logten,     # log(10)
     logπ,       # log(π)
     log2π,      # log(2π)
-    log4π       # log(4π)
+    log4π,      # log(4π)
+    invℯ,       # 1 / ℯ
```

Comment:
Maybe
```suggestion
    inve,       # 1 / e
```
would be easier? I didn't know how one can type `ℯ`, and I assume in general it's much less known than how to type `π`. Since `e` is the non-Unicode alias of `ℯ` in base, I think we could either just define `inve` or make it an alias of the Unicode version (out of the scope of this PR, but probably it would be good to add non-Unicode aliases for the other constants as well).

Reply:
I guess we can have both `inve` and `invℯ`, since `Base.MathConstants` also has both `e` and `ℯ`.